test/Network/IpAddress.cpp

```diff
@@ -0,0 +1,223 @@
+#include <SFML/Network/IpAddress.hpp>
+
+#include <doctest.h>
+
+#include <sstream>
+
+using namespace std::string_literals;
+
+TEST_CASE("sf::IpAddress class - [network]")
+{
+    SUBCASE("Construction")
+    {
+        SUBCASE("Default constructor")
+        {
+            const sf::IpAddress ipAddress;
+            CHECK(ipAddress.toString() == "0.0.0.0"s);
+            CHECK(ipAddress.toInteger() == 0);
+            CHECK(ipAddress != sf::IpAddress::Any);
```

Comment:
Because default-constructed IP addresses are invalid yet still report an address of 0.0.0.0. Maybe I should check equality with `sf::IpAddress::None` instead.

Reply:
> Maybe I should check equality with `sf::IpAddress::None` instead.

I think that would be good